cekit/log.py

```diff
@@ -26,7 +26,7 @@ def setup_logging():
     handler_err.addFilter(SingleLevelFilter(logging.INFO, True))
 
     formatter = colorlog.ColoredFormatter(
-        '%(log_color)s%(asctime)s %(name)-12s %(levelname)-8s %(message)s')
+        '%(log_color)s%(asctime)s %(filename)s:%(lineno)-10s %(levelname)-5s %(message)s')
```

Comment:
While I found this *very* useful I don't know if you want this reverted. It would be nice if it could be enabled perhaps if debug mode is enabled ?